service/frontend/templates/clusterredirection.tmpl

```diff
@@ -111,6 +113,9 @@ func (handler *clusterRedirectionHandler) {{$method.Declaration}} {
 	var actClSelPolicyForNewWF *types.ActiveClusterSelectionPolicy
 	var workflowExecution *types.WorkflowExecution
 	{{- if has $method.Name $startWFAPIs}}
+	if !handler.config.EnableActiveClusterSelectionPolicyInStartWorkflow(domainEntry.GetInfo().Name) {
+		{{(index $method.Params 1).Name}}.ActiveClusterSelectionPolicy = nil
```

Comment:
I don't love that htis is in the cluster-redirection handler, as it's nothing to do with redirection, but I guess there's nowhere better to put it :(